app.py

```diff
@@ -15,6 +19,12 @@ def create_app():
     # Load the configurations based on the 'FLASK_ENV' environment variable
     app.config.from_object(config)
 
+    # setup logging for access logs
+    handler = RotatingFileHandler('access.log', maxBytes=10000, backupCount=1)
+    handler.setFormatter(formatter())
+    handler.setLevel(logging.INFO)
```

Comment:
@Whazor this should be based on the DEBUG property of the flask run. If the DEBUG is true, this should be Debug else Info.